src/rules/no-misleading-boolean-prefixes.ts

```
@@ -0,0 +1,332 @@
+
+ const DEFAULT_PREFIXES = ['is', 'has', 'should'];
+
+ function isUppercaseLetter(char: string): boolean {
+   return char >= 'A' && char <= 'Z';
+ }
+
+ function isDigit(char: string): boolean {
+   return char >= '0' && char <= '9';
+ }
+
+ function startsWithBooleanPrefix(name: string, prefixes: string[]): boolean {
```

ESLint

[error] Delete `·`

(prettier/prettier)